tools/mo/openvino/tools/mo/moc_frontend/paddle_frontend_utils.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+
+def convert_paddle_to_pdmodel(model, inputs=None, outputs=None):
+    '''
+        There are three paddle model categories:
+        - High Level API: is a wrapper for dynamic or static model, use `self.save` to serialize
+        - Dynamic Model: use `paddle.jit.save` to serialize
+        - Static Model: use `paddle.static.save_inference_model` to serialize
+    '''
+    model_name = None
+    tmp = tempfile.NamedTemporaryFile(delete=True)
+    model_name = tmp.name
+    try:
+        import paddle
+        if isinstance(model, paddle.hapi.model.Model):
+            model.save(model_name, False)
+        else:
+            if inputs is None:
+                raise RuntimeError(
+                    "Saving inference model needs 'inputs' before saving. Please specify 'inputs'"
+                )
+            if isinstance(model, paddle.fluid.dygraph.layers.Layer): #paddle.in_dynamic_mode():
+                with paddle.fluid.framework._dygraph_guard(None):
+                    layer = model
+                    paddle.jit.save(model, model_name, input_spec=inputs)
+
+            elif isinstance(model, paddle.fluid.executor.Executor):
+                if outputs is None:
+                    raise RuntimeError(
+                        "Model is static. Saving inference model needs 'outputs' before saving. Please specify 'outputs'"
```

Comment:
Should user specify `output` or `example_output`? Not sure that `example_output is needed` because there already exists `output` option in MO

Reply:
> Should user specify `output` or `example_output`? Not sure that `example_output is needed` because there already exists `output` option in MO

Maybe not. Two reasons:
1. The `example_output` works like `example_input` in pytorch, which specifies the input and output tensor for the model saving.
2. It just works for `inline conversion` so it won't be involved in MO command options. 

Now,  it is alligned with `pytorch`. If we reuse `input` and `output` options in MO, the `example_input` should also be removed. So, could you please provide more comments here for better `User Experience`? @rkazants @popovaan

Reply:
@xczhai, I cannot understand why this parameter `example_output` is needed. It was a surprise for me to find such a parameter in `convert_model`, and I don't understand how it should be used. Please, provide specific examples.